src/main/java/io/vertx/core/impl/resolver/DnsResolverProvider.java

```diff
@@ -62,10 +62,10 @@
   private final Vertx vertx;
   private final List<ResolverRegistration> resolvers = Collections.synchronizedList(new ArrayList<>());
   private AddressResolverGroup<InetSocketAddress> resolverGroup;
+  public final List<InetSocketAddress> serverList = new ArrayList<>();
```

Comment:
what's the purpose of this ?

Reply:
that is where we are storing DNS server addresses

Reply:
so the first one will be used as a default one